lib/classifier-reborn/lsi/content_node.rb

```diff
@@ -47,7 +52,7 @@ def raw_vector_with( word_list )
       # Perform the scaling transform and force floating point arithmetic
       if $GSL
         sum = 0.0
-        vec.collect{|v| sum += v}
+        vec.each {|v| sum += v }
         total_words = sum
```

Comment:
It was slower than each, I benchmarked every change pretty thoroughly, but you are welcome to verify.